.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
**issue:** You are missing scripts specified in task requirements (but you can leave `npm test` as well):

```
When doing commit, Husky pre-commit hook should run yarn lint, and yarn pretty
```